### DISCUSSION TITLE
JSBSim launched from command line stop before simulation start with no message

### DISCUSSION THREAD

**Discussion by pbecchi** [JSBSim-Team/jsbsim#540] [General]
Hi I am new to JSBSim,

I have compiled and run few time from command line with no problem.
Yesterday I have installed Flightgear.
Today with the same line command used 2 days ago, JSBSim stop prior to start simulation with no message.

Can anybody help me?

**Comment by bcoconni** [comment]
FlightGear compiles its own version of JSBSim. May be you should check that the command line pick the right executable ?

**Comment by pbecchi** [comment]
I have found the reason ..... it is my fault . I have modified the code and the stop was due to a divide by zero exception.
Now the code run again ....I will send a message on #333 with results of my tryals.
Thanks anyhow for your support.
